src/main/java/pl/sda/transporeon/currencyexchange/controller/ExchangeRateController.java

```diff
@@ -17,15 +17,24 @@ public class ExchangeRateController {
 
     private final ExchangeRateService exchangeRateService;
     private final ExchangeRateRepository repository;
+    private final ExchangeStatisticRepository statisticRepository;
+    private final ExchangeStatisticService statisticService;
+    public static final String gold = "XAU";
+    public static final String pln = "PLN";
 
     @Autowired
-    public ExchangeRateController(ExchangeRateService exchangeRateService, ExchangeRateRepository repository) {
+    public ExchangeRateController(ExchangeRateService exchangeRateService, ExchangeRateRepository repository, ExchangeStatisticRepository statisticRepository, ExchangeStatisticService statisticService) {
         this.exchangeRateService = exchangeRateService;
         this.repository = repository;
+        this.statisticRepository = statisticRepository;
+        this.statisticService = statisticService;
     }
 
     @PostMapping("/exchange")
     ResponseEntity<ExchangeRateDTO> createRate(@RequestBody ExchangePost toCreate) {
+
+        statisticService.saveStatisticModelToBd(new ExchangeStatisticModel(toCreate.getBaseCurrency(), toCreate.getTargetCurrency(), toCreate.getExchangeDate()));
```

Comment:
saveStatisticModelToBd -> literówka (powinno być DB), ale to z jednej strony; z drugiej znowu nie ma sensu korzystać z nomenklatury techniczej (`saveStatistics` w zupełności wystarczy)